### PR TITLE
allow images of any ratio to be set as thumbnails without stretching them

### PR DESCRIPTION
**Pull request by mijorus**
the results can be seen on my website where I have deployed those changes (www.scifiostia.it):
images in the project section have different ratios but are now displayed correctly

### CODE REVIEW

File: assets/sass/_base.sass [hunk 1]
```diff
@@ -74,6 +74,11 @@ pre
 .thumbnail
     border: none
 
+.card-thumbnail
+  width: 100%	  width: 100%
+  height: 100%	  height: 100%
```
Comment (victoriadrake) [inline assets/sass/_base.sass:79]: Is this repetition intentional?

Comment (mijorus) [reply assets/sass/_base.sass]: nope, it was just a typo, probably when I merged my theme with your master branch. Should be fixed now

Comment (mijorus) [reply assets/sass/_base.sass]: do I have to send the PR once more? I have never edited a file after I sent a PR

Comment (victoriadrake) [reply assets/sass/_base.sass]: You should be able to push another commit to your branch, and this PR will account for it.

**Comment by victoriadrake** [comment]
Thank you @mijorus!

**Pull request merged**